server.js

```diff
@@ -257,6 +257,12 @@ app.use('/pl/administrator/overview', require('./pages/administratorOverview/adm
 //////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////
+// Webhooks //////////////////////////////////////////////////////////
+app.use('/pl/webhooks/autograder', require('./webhooks/autograder/autograder'));
```

Comment:
I agree on the general separation of concerns question. However, it's only one path (webooks) that needs to be special-cased at the moment, so hacky solutions appeal to me rather than more complicated general frameworks.